rust-runtime/aws-smithy-http/Cargo.toml

```diff
@@ -27,30 +27,20 @@ pin-project-lite = "0.2.9"
 pin-utils = "0.1.0"
 tracing = "0.1"
 
-# We are using hyper for our streaming body implementation, but this is an internal detail.
-hyper = "0.14.26"
-
-# ByteStream internals
+# For our event stream implementation
 futures-core = "0.3.14"
-tokio = { version = "1.23.1", optional = true }
-tokio-util = { version = "0.7", optional = true }
+hyper = "0.14.26"
```

Comment:
Can hyper be an optional dependency now? If so, we should add a `hyper014` feature.

Reply:
Made it an optional dependency in aae39be